### PR TITLE
DOC Expand intro in readme

### PR DESCRIPTION
**Pull request by lucyleeow**
I know this is WIP, but thought we could expand the description to provide more info. Might have been a bit confusing, especially to newcomers. Happy to change wording.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 [![tests](https://github.com/napari/napari-plugin-manager/workflows/test_and_deploy/badge.svg)](https://github.com/napari/napari-plugin-manager/actions)
 [![codecov](https://codecov.io/gh/napari/napari-plugin-manager/branch/main/graph/badge.svg)](https://codecov.io/gh/napari/napari-plugin-manager)
 
-A plugin that adds a plugin manager to [napari].
+[napari] plugin manager to enable installing [napari] plugins inside the napari application, that is itself a plugin.
```
[the remark this answers is not in the document]
Comment (lucyleeow) [reply README.md:11]: Much better, thank you.

**Review by psobolewskiPhD — APPROVED** [review]

**Review by goanpeca — APPROVED** [review]

**Comment by Czaki** [comment]
Why is it important to highlight that it is a plugin itself?

**Comment by jaimergp** [comment]
Actually it's not really a plugin after all (it might become one, but it's not right now).

**Comment by lucyleeow** [comment]
The aim was not to highlight that it is a plugin, it was to clarify the original sentence which also said it was a plugin. However as pointed out it currently is not so will amend.

**Comment by lucyleeow** [comment]
Done, thanks all

**Review by jaimergp — APPROVED** [review]

**Review by goanpeca — APPROVED** [review]

**Pull request merged**